### PR TITLE
Paginated RPC pay.operations

### LINKED ISSUE #2402: Implement pagination for getting incomes
We may don't want to get list of all incomes in UI, just receive them in batches. 

Depends on:
* [ ] Generic pagination mechanism for RPC https://github.com/golemfactory/golem/issues/3970

### LINKED ISSUE #3971: Implement pagination for outgoing payments
Depends on:
* [ ] Generic pagination mechanism for RPC https://github.com/golemfactory/golem/issues/3970

### PR DESCRIPTION
**Pull request by jiivan**
Part of #4172
Fixes: #2402  
Fixes: #3971

### CODE REVIEW

**Comment by etam** [comment]
Is it related to any of #3971, #3970, #2402?

**Comment by jiivan** [comment]
> Is it related to any of #3971, #3970, #2402?

Actually to #4172 (so yes - to all of the above)

**Comment by jiivan** [comment]
> Is it related to any of #3971, #3970, #2402?

It will somehow fix #3971 and #2402 but not #3970

File: tests/golem/ethereum/test_paymentprocessor.py [hunk 1]
```diff
@@ -484,6 +485,8 @@ def add_payment(self, processed_ts: int):
             wallet_operation__direction=  # noqa
             model.WalletOperation.DIRECTION.outgoing,
             created_date=timestamp_to_datetime(processed_ts),
+            wallet_operation__status=  # noqa
+            model.WalletOperation.STATUS.awaiting,
```
Comment (etam) [inline tests/golem/ethereum/test_paymentprocessor.py:489]: this could be a one line, like above

File: golem/rpc/api/ethereum_.py [hunk 2]
```diff
@@ -72,6 +74,71 @@ def item(o):
...
+            query = query.where(
+                model.WalletOperation.operation_type == operation_type,
+            )
+        query = query.paginate(page_num, per_page)
+        tp_query = model.TaskPayment.select() \
+            .where(
+                model.TaskPayment.wallet_operation.in_(query),
+            )
+        task_payments_map = dict(
+            (tp.id, tp) for tp in tp_query
```
Comment (etam) [inline golem/rpc/api/ethereum_.py]: you can also write
```python
task_payments_map = {tp.id: tp for tp in tp_query}
```

**Review by etam — APPROVED** [review]
lgtm

File: tests/golem/ethereum/test_paymentprocessor.py (hunk 1 above)
Comment (etam) [reply tests/golem/ethereum/test_paymentprocessor.py:489]: (I mean in previous change, in line 66)

**Pull request merged**